definition/IRoom.ts

```diff
@@ -160,6 +160,26 @@ export interface IOmnichannelRoom extends Omit<IRoom, 'default' | 'featured' | '
 	ts: Date;
 	label?: string;
 	crmData?: unknown;
+
+	// optional keys for closed rooms
+	closer?: 'user' | 'visitor';
+	closedBy?: {
+		_id: string;
+		username: IUser['username'];
+	};
+	serviceTimeDuration?: string;
+	extraData?: Record<string, any>;
```

Comment:
```suggestion
```
extraData is not part of the room.

Reply:
@KevLehman Any specific reason to add this?

Reply:
Ooops, this has been around more than it should 😬 let me remove it